contracts/SignerBondsEscrow.sol

```diff
@@ -28,9 +28,19 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 ///         signer bonds so that governance can later swap them manually and
 ///         deposit as coverage pool collateral.
 contract SignerBondsEscrow is ISignerBondsSwapStrategy, Ownable {
-    /// @notice Swaps signer bonds.
+    /// @notice Receive ETH upon withdrawal of risk manager's signer bonds.
+    receive() external payable {}
+
+    /// @notice Swaps the given signer bonds amount from the given risk manager.
     /// @dev Adds incoming bonds to the overall contract balance.
-    function swapSignerBonds() external payable override {}
+    /// @param riskManager Address of the risk manager
+    /// @param amount Amount of signer bonds being swapped.
+    function swapSignerBonds(RiskManagerV1 riskManager, uint256 amount)
```

Comment:
Do we need this function? Wouldn't it be better to call `riskManager.withdrawSignerBonds(amount);` as a part of `withdraw` and modify `withdraw` signature so that it accepts `amount` as well? And even more high level question - do we still need `SignerBondsEscrow` or maybe `SignerBondsManualSwap` next to `SignerBondsUniswapV2`? Historically, `SignerBondsEscrow` was indeed an escrow funded with each purchased signer bonds. Now, those bonds are deposited on the risk manager, until a strategy decides to withdraw them.

Right now, withdrawing bonds requires two transactions and I do not see a use case for withdrawing bonds from the risk manager but not withdrawing them from the escrow contract and accumulating them in the escrow.

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/88/commits/874455749efe522f06e634c8e36ef977a4a3cb44